### PR TITLE
Nightly integration tests on Travis and Security Group Changes.

### PR DESCRIPTION
**Pull request by TheKLARKEN**
This adds changes to the Security Groups rules set on kube creation (AWS). Also, this adds a single line to keep integration tests from running during every single PR or push. They'll only run nightly during cron jobs.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -59,6 +59,9 @@ script:
   - echo 'mode':' count' | cat - .coverprofile > /tmp/out && mv /tmp/out .coverprofile
   - goveralls -coverprofile=.coverprofile -service=travis-ci
 
+  # Integration
+  - if [ "$TRAVIS_EVENT_TYPE" == "cron" ] ; then test -tags=integration ./pkg/... ./test/... ; fi
```
Comment (saheienko) [inline .travis.yml]: go test?
And, it is better to add '-v' flag for such kind of tests.

**Comment by stgleb** [comment]
LGTM

File: .travis.yml (hunk 1 above)
Comment (stgleb) [reply .travis.yml]: Also, those integration test that I've written need a bunch of env variables to be set.

For instance

https://github.com/stgleb/supergiant/blob/providers_versions/test/integration/providers/aws_test.go#L25

Comment (TheKLARKEN) [reply .travis.yml]: Are these the only variables that need to be set? You can set them at https://travis-ci.org/supergiant/supergiant/settings if you'd like. Or I can set them if you'd like me to.

**Comment by TheKLARKEN** [comment]
Didn't realize my commits were automatically added to this PR. 😅 Things should be solid now!

**Comment by yegorch** [comment]
@TheKLARKEN tests are failing. Can you please rebase and fix them?

**Comment by TheKLARKEN** [comment]
This is killing me. `unable to open database file` is produced by returning `SQLITE_CANTOPEN`, and that exact error is returned in almost 30 different places under different circumstances, and I can't tell at all which one is happening and, therefore, why. It's for an old integration test Ben wrote, so I can't ask him for help, either. 😕 
```
--- FAIL: TestCloudAccountsList (0.00s)
panic: unable to open database file [recovered]
	panic: unable to open database file

goroutine 40 [running]:
testing.tRunner.func1(0xc4205460f0)
	/usr/local/Cellar/go/1.10.2/libexec/src/testing/testing.go:742 +0x29d
panic(0x508daa0, 0xc420148c20)
	/usr/local/Cellar/go/1.10.2/libexec/src/runtime/panic.go:502 +0x229
github.com/supergiant/supergiant/test/api.wipeAndInitialize(0xc4205a4600)
	/Users/clarkevennerbeck/go/src/github.com/supergiant/supergiant/test/api/testutils_test.go:51 +0x66
github.com/supergiant/supergiant/test/api.newTestServer(0x5af7317)
	/Users/clarkevennerbeck/go/src/github.com/supergiant/supergiant/test/api/testutils_test.go:19 +0xe6
github.com/supergiant/supergiant/test/api.TestCloudAccountsList(0xc4205460f0)
	/Users/clarkevennerbeck/go/src/github.com/supergiant/supergiant/test/api/cloud_accounts_test.go:15 +0x26
testing.tRunner(0xc4205460f0, 0x521eb20)
	/usr/local/Cellar/go/1.10.2/libexec/src/testing/testing.go:777 +0xd0
created by testing.(*T).Run
	/usr/local/Cellar/go/1.10.2/libexec/src/testing/testing.go:824 +0x2e0
FAIL	github.com/supergiant/supergiant/test/api	0.054s
Error: Tests failed.
```

**Comment by TheKLARKEN** [comment]
It worked! I completely forgot that I moved a folder, and that's what broke it! Thanks for your help, everyone!

**Pull request merged**